accession-commons-monotonic-generator-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/generators/monotonic/MonotonicAccessionGenerator.java

```diff
@@ -107,15 +101,16 @@ public synchronized void recoverState(long[] committedElements) throws Accession
     private synchronized void reserveNewBlocksUntilSizeIs(int totalAccessionsToGenerate) {
         while (!blockManager.hasAvailableAccessions(totalAccessionsToGenerate)) {
             try {
-                ExponentialBackOff.execute(() -> reserveNewBlock(categoryId, applicationInstanceId, blockSize), 10, 30);
+                blockService.checkIsBlockInitializationsValid(categoryId);
```

Comment:
Not needed if next comment is applied.

Reply:
Has this comment been addressed or is it no longer applicable?

Reply:
Its no longer applicable because , we need to check the block parameters before calling ExponentialBackoff, Jorge suggested to add this check in blockservice constructor , but we will not be aware of the category we gonna use on that stage, so explicitly doing the check depending on the category here.

Reply:
I think I suggested to do it in the initialization of the properties. Let me recheck the code because it seems something has slipped me.